logslurp.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+if [[ -z $1 ]] || [[ -z $2 ]]; then
+    echo "Usage: $0 adminuser@master adminuser@windowsnode"
+    echo
+    echo "Note: SSH agent forwarding must be enabled for your current session to avoid passwords"
+    exit 1
+fi
+
+read -r -d '' remoteCommand <<'EOF'
+    $lockedFiles = "kubelet.err.log", "kubelet.log", "kubeproxy.log", "kubeproxy.err.log"; 
```

Comment:
Should this be in a single shared file between the win-rm and bash scripts so they don't diverge?  This would also add the ability to download the script and run it adhoc if need be.

Reply:
Yeah, that would be ideal. I'd probably need to add a Makefile to build a fully-formed bash script that we can curl and run as well as the container version.